src/wp-admin/js/customize-controls.js

```diff
@@ -8811,7 +8856,6 @@
 		// Bind site title display to the corresponding field.
 		if ( title.length ) {
 			api( 'blogname', function( setting ) {
-				// eslint-disable-next-line func-style
```

Comment:
`// eslint-disable-next-line func-style`

Leave this comment line in the code to silence `Expected a function declaration` errors / warnings in ESLint.

Reply:
Thanks for the explanation! Now done.